component/src/main/java/io/siddhi/extension/store/rdbms/metrics/RDBMSStatus.java

```diff
@@ -0,0 +1,9 @@
+package io.siddhi.extension.store.rdbms.metrics;
```

Comment:
Add license headers to the file

Reply:
Resolve in bf12c58e32ebfc0ebc0b9ccad12962e1f0218070